assets/src/components/cd/services/ServiceDependenciesColumns.tsx

```diff
@@ -0,0 +1,27 @@
+import { ServiceDependency } from 'generated/graphql'
+import { createColumnHelper } from '@tanstack/react-table'
+import { Chip } from '@pluralsh/design-system'
+
+const columnHelper = createColumnHelper<Nullable<ServiceDependency>>()
+
+const ColName = columnHelper.accessor('name', {
+  id: 'name',
+  header: 'Name',
+})
+
+const ColStatus = columnHelper.accessor('status', {
+  id: 'status',
+  header: 'Status',
+  cell: ({ getValue }) => (
+    <div>
+      <Chip
+        size="large"
+        severity={getValue() === 'HEALTHY' ? 'success' : 'warning'}
```

Comment:
ideally we use the actual codegened enum here, but can't remember how to import it

Reply:
yeah good call. typescript was still throwing the warning if the checked string wasn't part of the enum but I went ahead and imported it